testomania/src/main/res/values/strings.xml

```diff
@@ -19,6 +19,7 @@
     <string name="navigation_next">Next/Skip</string>
     <string name="coming_soon">Coming soon...</string>
     <string name="dismiss">Dismiss</string>
+    <string name="back">Back</string>
```

Comment:
I think this is not used. Do we need this string?

Reply:
Yes, this is not used, I'll remove it.